src/test/java/org/cloudfoundry/promregator/cache/AutoRefreshingCacheMapTest.java

```diff
@@ -265,13 +269,15 @@ public boolean equals(Object obj) {
 			if (a == null) {
 				if (other.a != null)
 					return false;
-			} else if (!a.equals(other.a))
+			} else if (!a.equals(other.a)) {
```

Comment:
Just for the record: I understand what's up here, but we will not have fun with it: These kind of methods are auto-generated by Eclipse.... So, as soon as we regenerate them, we have to fix the SonarQube Recommendation again.

Reply:
Does Eclipse have updated templates that dont upset Sonar?

Reply:
Not updated, but you may configure it: When generating set the flag "use blocks in 'if' statements".

Sounds like something that we should write down in some sort of project style guidelines...(I will take care)

Reply:
How do you wish to proceed?

Reply:
done with cd744f0503c3660621b968bc0f71f3edba6cda95

Reply:
> How do you wish to proceed?

Let's merge your stuff. The [new style guide](https://github.com/promregator/promregator/blob/81078620d64fd33bdad1255acfa08f4b2e528403/CONTRIBUTING.md) should make sure that future usage of the Eclipse generator will not break it anymore.